### PR TITLE
fix(modal): mount nested modals inside global modals

### LINKED ISSUE #11543: Conversation Setting, Advanced Permissions pop-up screen hidden
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Create a group call
2. Select 'Conversation Settings'
3. Go to the section 'Permissions'
4. Try to edit advanced permission settings (popup screen is hidden behind the first pop-up screen)

### Expected behaviour
The pop-up screen to edit adavnced permission settings should be visible to the user

### Actual behaviour
When clicking the 'pencil' to edit advdance settings, the pop up screen shows up behind the already opened screen

## Talk app

**Talk app version:** (see apps admin page: `/index.php/settings/apps`)
Talk Version 18.0.3

**Custom Signaling server configured:** yes/no and version (see additional admin settings: `/index.php/index.php/settings/admin/talk#signaling_server`)

**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#turn_server`)

**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#stun_server`)

## Browser
Checked the issue with Chrome, FireFox and Edge (all latest version as time of writting)

**Microphone available:** yes/no

**Camera available:** yes/no

**Operating system:** Windows/Ubuntu/...
On windows and Linux: Same issues.

**Browser name:** Firefox/Chrome/...

**Browser version:** 85/96/...

### Browser log

<details>
```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

## Server configuration

**Operating system**: Ubuntu/RedHat/...

**Web server:** Apache/Nginx
Nextcloud instance is installed on Apache

**Database:** MySQL/Maria/SQLite/PostgreSQL
Mariadb

**PHP version:** 8.0/8.1/8.2
PHP 8.1

**Nextcloud Version:** (see admin page)
Nexcloud 28.0.2

**List of activated apps:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```
</details>

**Nextcloud configuration:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Server log (data/nextcloud.log)
<details>

```
Insert your server log here
```
</details>

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix #11543
  * Mount Advanced settings inside the Conversation settings to ensure it appears above
  * Hard to reproduce initial issue, but this should exclude possibility of falling behind
  * Also added a container prop for MessageTranslateDialog and PollVotersDetails

- [ ] TODO: check NcDialog components and wrappers

### 🖼️ Screenshots / Screencasts

| 🏚️ Before | 🏡 After |
|------------|----------|
| image          | image        |

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/ConversationSettings/DangerZone.vue [hunk 1]
```diff
@@ -51,7 +51,7 @@
 					:open.sync="isDeleteConversationDialogOpen"
 					:name="t('spreed','Delete Conversation')"
 					:message="deleteConversationDialogMessage"
-					:container="container">
+					container="#settings-section_dangerzone">
```
Comment (DorraJaouad) [inline src/components/ConversationSettings/DangerZone.vue:54]: It's not critical, but I would recommend to use our own classes and not depend on upstream ids.

Comment (Antreesy) [reply src/components/ConversationSettings/DangerZone.vue:54]: Fair

**Review by DorraJaouad — APPROVED** [review]
No other modal remaining ✅ .

**Pull request merged**